specs/text-area/text-area.md

```diff
@@ -0,0 +1,130 @@
+- [Evergreen](https://evergreen.segment.com/components/textarea/)
+
+---
+
+### API
+
+*Component Name*
+- `fast-text-area`
+
+*Attrs* - Adapted from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea)
+- `appearance` - enum
+  - outline (default)
+  - filled
+- `autofocus` -  Boolean attribute lets you specify that a form control should have input focus when the page loads
+- `cols` - visible width of the text control, in average character widths. If it is specified, it must be a positive integer. If it is not specified, the default value is 20
+- `disabled` - disables the control
+- `form` - form element that the `<textarea>` element is associated with (its "form owner"). The value of the attribute must be the id of a form element in the same document. If this attribute is not specified, the `<textarea>` element must be a descendant of a form element. This attribute enables you to place `<textarea>` elements anywhere within a document, not just as descendants of form elements
+- `list` - the id of the `<datalist>` element that contains the optional pre-defined autocomplete options
+- `maxlength`	- the maximum number of characters the input should accept
+- `minlength` -	the minimum number of characters long the input can be and still be considered valid
+- `name` - the name of the control
+- `placeholder` - an exemplar value to display in the `<textarea>` whenever it is empty
+- `readonly` - the `<textarea>` should be submitted with the form but should not be editable.
+- `required` - boolean value that sets the `<textarea>` as required
+- `rows` - the number of visible text lines
+- `spellcheck` - controls whether or not to enable spell checking for the `<textarea>`, or if the default spell checking configuration should be used
+
+*Events*
+- `change: CustomEvent`
+  - no custom data
+  - bubbles
+
+### Anatomy and Appearance
+
+```HTML
+<!-- shadow root -->
+<label part="label">
+    <slot name="label"></slot>
+</label>
+<textarea
+    part="control"
+    $autofocus="${x => x.autofocus}"
+    $cols="${x => x.cols}"
+    $disabled="${x => x.disabled}"
+    $form="${x => x.form}"
+    $list="${x => x.list}"
+    $maxlength="${x => x.maxlength}"
+    $minlength="${x => x.minlength}"
+    $name="${x => x.name}"
+    $placeholder="${x => x.placeholder}"
+    $readonly="${x => x.readonly}"
+    $required="${x => x.required}"
+    $rows="${x => x.rows}"
+    $spellcheck="${x => x.spellcheck}"
+>
+    <slot></slot>
+</textarea>
+<!-- end shadow root -->
+```
+
+
+*Screenshots below are of the basic appearance of the component and are not exhaustive.*
+
+| State | Image |
+| ----- | ----- |
+| default | ![](./images/text-area.png) |
+| focus | ![](./images/text-area-focus.png)
+| placeholder | ![](./images/text-area-placeholder.png)
+
+*Slot Names*
+- default - the content of the text area. Unlike input where content is a value, the content of a text area is predefined children.
+- label - the label content
```

Comment:
Should the images show a one with a label?

Reply:
We can - I don't technically have an image for that or filled...so I didn't include those. Happy to though.

Reply:
added